tests/test_ms2library.py

```diff
@@ -1,17 +1,88 @@
 import os
 import numpy as np
+import pytest
 from pandas.testing import assert_frame_equal
 from matchms import Spectrum
 from ms2query.ms2library import MS2Library, get_ms2query_model_prediction
 from ms2query.app_helpers import load_pickled_file
 from pandas import DataFrame
 
 
-def test_ms2library_set_settings():
+@pytest.fixture
```

Comment:
Nice! Was planning to do this here as well, after learning that this was an option, but forgot, I will make sure to use it for new tests!